docs/workflows/copy.rst

```diff
@@ -0,0 +1,35 @@
+Copy RPM content between repositories
+=====================================
+
+If you want to copy all of the RPM content in one repository into another repository, you can do so.
+
+
+.. _copy-workflow:
+
+Copy workflow
+-------------
+
+You can use copy endpoint to copy all of the RPM-related content present in one repository into another repository.
+You can specify either a repository version to copy from, or a repository (in which case, the latest repository
+version for that repository will be used).
+
+``http POST http://localhost:24817/pulp/api/v3/rpm/copy/ source_repo=${SRC_REPO_HREF} dest_repo=${DEST_REPO_HREF}``
+
+or
+
+``http POST http://localhost:24817/pulp/api/v3/rpm/copy/ source_repo_version=${SRC_REPO_VER_HREF} dest_repo=${DEST_REPO_HREF}``
+
+.. code:: json
+
+    {
+       "_href": "/pulp/api/v3/tasks/f2b525e3-8d8f-4246-adab-2fabd2b089a8/",
+       "created_resources": [
+           "/pulp/api/v3/content/rpm/packages/1edf8d4e-4293-4b66-93cd-8e913731c87a/",
+       ],
+
+       ...
+    }
+
+You can also specify which types of content you would like to copy by providing a value for the "types" parameter. Types that are not listed will not be copied. The supported types are "packages" and "errata". For example, this query will copy only errata, and not packages.
+
+``http POST http://localhost:24817/pulp/api/v3/rpm/copy/ source_repo=${SRC_REPO_HREF} dest_repo=${DEST_REPO_HREF} types:="[\"errata\"]"``
```

Comment:
do make the connection of `errata` and `update record` terminology in our docs? users copy errata and in the repo version summary they see instead update record. I am wondering - won't it be better if we just stick to `update record`?
this reminds me of our bitter experience in pulp2 about tasks state naming failed vs errored, sucessful vs finished.

Reply:
nevermind, the endpoint for errata is `/pulp/api/v3/content/rpm/errata/` so i guess it's ok

Reply:
@dralley any reason we did not go with comma separated values? `types=rpm,errata`

Reply:
@ipanova only because this is the way things were already being done for 
"add_content_units="

Reply:
@ipanova Personally I think we should also change the _type string so that they show up as errata in the repository version also.  The only place I've ever seen the update record terminology used is in the createrepo_c source code - I can't seem to find any other documentation calling it that vs. errata.  So I don't think it's a secret "official" name vs. a marketing term or anything like that.